crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -302,7 +310,21 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for MultisafepayPaymentsReques
                 moto: None,
                 term_url: None,
                 email: None,
+                payment_token: None,
             },
+            api::PaymentMethodData::Wallet(api::WalletData::GooglePay(ref google_pay)) => {
+                GatewayInfo {
+                    card_number: None,
+                    card_holder_name: None,
+                    card_expiry_date: None,
+                    card_cvc: None,
+                    flexible_3d: None,
+                    moto: None,
+                    term_url: None,
+                    email: None,
```

Comment:
Will connector throw error in production if these fields are not provided?
can we get it clarified from their docs or customer support pls

Reply:
I mailed their support, as per their response they will not fire an error

Reply:
@AkshayaFoiger Lets changes GatewayInfo to enum which holds 2 values. 1 for cards another for GPay, each will have it's own struct.